packages/superset-ui-chart/src/index.ts

```diff
@@ -21,6 +16,10 @@ export {
   default as getChartTransformPropsRegistry,
 } from './registries/ChartTransformPropsRegistrySingleton';
 
-export { QueryContext, buildQueryContext } from './query/buildQueryContext';
-export { DatasourceType, DatasourceKey } from './query/DatasourceKey';
-export { FormData } from './query/FormData';
+export { default as buildQueryContext } from './query/buildQueryContext';
+export { default as DatasourceKey } from './query/DatasourceKey';
+
+export * from './types/Annotation';
```

Comment:
generally I think it's good to be explicit with exports for readability + ensuring that there is no conflict / overriding of variables (unsure if that would error or cause subtle bugs for types)